src/api/public/apidocs-new/components/parameters/search_limit.yaml

```diff
@@ -0,0 +1,6 @@
+in: query
+name: limit
+schema:
+  type: integer
+description: TODO
```

Comment:
To do or not to do... this is the question! :smile:

Same thing for the file *src/api/public/apidocs-new/components/parameters/search_offset.yaml*. Both are missing a description.

Reply:
Ops, true. Fixed.